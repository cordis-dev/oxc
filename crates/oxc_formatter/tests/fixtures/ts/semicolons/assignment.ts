(<T>pendingSetRef.flags) |= SchedulerJobFlags.DISPOSED;
(pendingSetRef.flags as T) |= SchedulerJobFlags.DISPOSED;
(pendingSetRef.flags satisfies T) |= SchedulerJobFlags.DISPOSED;
(pendingSetRef.flags!) |= SchedulerJobFlags.DISPOSED;