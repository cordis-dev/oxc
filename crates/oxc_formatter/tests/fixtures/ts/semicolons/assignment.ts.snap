---
source: crates/oxc_formatter/tests/fixtures/mod.rs
---
==================== Input ====================
(<T>pendingSetRef.flags) |= SchedulerJobFlags.DISPOSED;
(pendingSetRef.flags as T) |= SchedulerJobFlags.DISPOSED;
(pendingSetRef.flags satisfies T) |= SchedulerJobFlags.DISPOSED;
(pendingSetRef.flags!) |= SchedulerJobFlags.DISPOSED;
==================== Output ====================
--------------
{ semi: true }
--------------
(<T>pendingSetRef.flags) |= SchedulerJobFlags.DISPOSED;
(pendingSetRef.flags as T) |= SchedulerJobFlags.DISPOSED;
(pendingSetRef.flags satisfies T) |= SchedulerJobFlags.DISPOSED;
pendingSetRef.flags! |= SchedulerJobFlags.DISPOSED;

---------------
{ semi: false }
---------------
;(<T>pendingSetRef.flags) |= SchedulerJobFlags.DISPOSED
;(pendingSetRef.flags as T) |= SchedulerJobFlags.DISPOSED
;(pendingSetRef.flags satisfies T) |= SchedulerJobFlags.DISPOSED
pendingSetRef.flags! |= SchedulerJobFlags.DISPOSED

===================== End =====================
